src/index.html

```diff
@@ -11,7 +11,165 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="mb-20">
+        <legend>Personal information</legend>
+        <div class="mb-10">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
+        </div>
+        <div class="mb-10">
+        Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            minlength="1"
+            maxlength="2"
+          >
+        </div>
+        <div class="mb-10">
+        How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+            >
```

Comment:
```suggestion
          >
```
[chacklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
![image](https://user-images.githubusercontent.com/99652389/178550776-489ded50-1065-470f-90fb-234f6c834f96.png)